RDS.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_db_instance" "takeon" {
```

Comment:
Shouldn't this be parameterised? db instance name reflecting the environment? I know the tag is but probably the name too?  Or is it covered by the name parameter from the vars file?

Reply:
These are just resource names, it could be changed to something more generic like RDS, the actual name of the db is one of the build parameters covered by the vars